runtime/crab/Cargo.toml

```diff
@@ -124,8 +124,14 @@ frame-system-benchmarking = { workspace = true, optional = true }
 frame-try-runtime         = { workspace = true, optional = true }
 
 [dev-dependencies]
+# crates.io
+static_assertions = { version = "1.1.0" }
```

Comment:
This can be added to workspace dependencies.

Reply:
Nice catch. This is already added in the workspace dependencies and also in the `[dependencies]` list of runtime. I think it can be deleted.